conf/ocs_version/ocs-4.5.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 DEPLOYMENT:
-  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-olm-operator:latest-4.5"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-bundle-operator:latest-4.5"
```

Comment:
The bundle image is not deployable itself. Per Boris Ranto's comment you need to use:
quay.io/rhceph-dev/ocs-registry:4.5.0-30.ci

Index image for deployment.

Reply:
Yes, OCP does not support the deployment of bundle images. Apparently, it was discussed but it was not included in the release.